src/App/App.js

```diff
@@ -0,0 +1,32 @@
+import { Routes, Route } from "react-router-dom";
+import "./App.css";
+import { Home } from "../pages/Home/Home";
+import {MockbeeHome} from "../pages/mockApi/MockbeeHome/MockbeeHome"
+import { MockmanComponent } from "../pages/mockApi/Mockman/MockmanComponent";
+import { Nav } from "../components/Nav/Nav";
+import { Cart } from "../pages/Cart/Cart";
+import { Wishlist } from "../pages/Wishlist/Wishlist";
+import { Auth } from "../pages/Authentication/Auth";
+import { ErrorPage } from "../pages/Error/ErrorPage";
+import { Footer } from "../components/Footer/Footer";
```

Comment:
you can export the files from the `index.js` file of your folder so that you don't need to write the whole path of the file to import it. ex- in the pages folder you can create an `'index.js` file then export these functions like `export {Nav} from "./Nav/Nav"` and then import them like `import {Nav, Cart, Wishlist } from "../pages"`. This is because whenever we try to import something from a folder, the bundler looks for an index.js file in the folder first.

Reply:
Yes this would reduce your code and its helpful.